### DISCUSSION TITLE
Limit the total number of ids

### DISCUSSION THREAD

**Discussion by WindnSnow** [mikel-brostrom/yolo_tracking#811] [Q&A] [answered]
I would like to know if it is possible to specify the maximum number of tracked IDs in a video? I have seen that there is `max-det` for the total number per image, but I would like to specify the maximum number of IDs (not detections) for the entire video. For instance, if I detect individuals and I know in advance that there are a total of 5 individuals, the IDs will not go above 5 for the entiere video. Thank you.

**Comment by mikel-brostrom** [comment, marked as answer]
This is not implemented but would be relatively easy. Count the tracks:

https://github.com/mikel-brostrom/yolov8_tracking/blob/1280ca5a77b3c2cd1bfa3e5b061a27e79f5091d2/trackers/strongsort/sort/tracker.py#L50

 and don't spawn new ones if count exceeds X

**Reply by WindnSnow** [reply]
I will check, thank you for your quick answer.
